### DISCUSSION TITLE
Rayyan's use of non-public user generated content - do you have more information?

### DISCUSSION THREAD

**Discussion by JonnaLind** [asreview/asreview#1534] [Q&A]
Hi Rens,

Your presentation of ASReview on the ARQ conference last week generated a lot of enthusiasm with my colleagues. One of them mentioned to me, that you mentioned that using Rayyan is not safe, as they are commercial and sell your data to third parties. Could you tell me something more about that? I don't see it in their Terms of service. I would be very interested in hearing more about this!

Met vriendelijke groet,

Jonna Lind

**Comment by Rensvandeschoot** [comment]
Thank you for your interest in my presentation at the ARQ conference. I apologize for any confusion. I did not intend to say that Rayyan is unsafe. My point was to emphasize the importance of being aware that Rayyan stores and uses your data, as stated in their privacy policy. You can read more about it [here](https://www.rayyan.ai/privacy_policy/). I mentioned this because some users may not be fully aware of these terms and may not want their data to be collected.
